templates/.env.j2

```diff
@@ -50,3 +54,4 @@ LETSENCRYPT_STAGING={{ docker_letsencrypt['staging'] }} # Set to 1 if you're tes
 DOCKER_PROXY_NGINX_TEMPLATE=scalelite-proxy
 #DOCKER_PROXY_NGINX_TEMPLATE=scalelite-local
 #DOCKER_PROXY_NGINX_TEMPLATE=scalelite-cluster
+{% endif %}
```

Comment:
I am unsure what `bbb_turn_enable` does.
Yes I see it's used in the template but is this related to coturn or what?